package.json

```
@@ -22,7 +22,7 @@
   "author": "Angel Nikolov <darkysharky@gmail.com>",
   "license": "ISC",
   "peerDependencies": {
-    "rxjs": "^6.3.0"
+    "rxjs": "^7.4.0"
```

```suggestion
    "rxjs": "^6.6.0 || ^7.4.0"
```
Hey thanks for the PR! We wouldn't want to make the library unusable for people using Angular versions so below 12 based on this matrix https://gist.github.com/LayZeeDK/c822cc812f75bb07b7c55d07ba2719b3.
We should probably put the versions to both 6.6.0 and 7.4.0.
Also, can you please run npm install and also commit the lock file?
Thanks!

---

Did the suggested change